Assets/Python/GreatPeople.py

```diff
@@ -275,40 +336,70 @@ def setup():
 		"Jamshetji Tata", # 19th
 		"Ardeshir Godrej", # 19th
 		iGlobal,
+		"V. O. Chidambaram Pillai", # 20th
```

Comment:
Can you find a form of the name that isn't too long without initials?

Reply:
Valliappan Olaganathan Chidambaram Pillai is a little long, isn't it? Apparently he's also known as Kappalottiya Tamizhan, "The Tamil Helmsman", so I'll put that.